discovery-provider/src/utils/helpers.py

```diff
@@ -542,3 +542,8 @@ def get_final_poa_block() -> int:
         raise Exception("audius_final_poa_block not set")
 
     return final_poa_block
+
+
+def format_total_audio_balance(balance: str) -> str:
+    balance_float = float(balance) / 1e18
```

Comment:
nit: could just use `int` here, no reason for it to be a float. if we wanted to be clever, the whole thing could be `return int(int(balance) / 1e18)`